docs/source/developer/interface/environment-variables.md

```diff
@@ -2,10 +2,29 @@
 
 You can alter Interface's functionality using these environment variables.
 
-Uses the source files for the QML and other resource at runtime instead of the compiled files.
+___
+
+Uses the source files for the QML and other resources at runtime instead of the compiled files.
 
     HIFI_USE_SOURCE_TREE_RESOURCES=1
 
+___
+
 Enables the OpenVR plugin to be used with an Oculus HMD. (Useful for debugging the OpenVR plugin if you have an Oculus HMD but not a Vive.)
 
     HIFI_DEBUG_OPENVR=1
+
+___
+
+Logging on the console can be controlled with the `VIRCADIA_LOG_OPTIONS` variable. It can be set to any combination of the following:
```

Comment:
```suggestion
Logging to the console can be controlled with the `VIRCADIA_LOG_OPTIONS` variable. It can be set to any combination of the following:
```
`Logging on the console` sounds weird to me. Normally `on` would refer to the physical machine or furniture. I would suggest `Logging to the console` as Vircadia is sending messages to the standard output.